docs-source/content/userguide/managing-domains/fmw-infra/_index.md

```diff
@@ -0,0 +1,353 @@
+commands we need to run. 
+
+To get inside this container and run commands, use this command: 
+
+```bash
+kubectl exec -ti rcu /bin/bash
+```
+
+When you are finished with this pod, you can remove it with this command: 
+
+```bash
+kubectl delete deploy rcu 
+```
+
+{{% notice note %}}
+You can use the same approach to get a temporary "service" pod to run other utilities
+like WLST.
+{{% /notice %}}
+
+##### Creating schemas
+
+Inside this pod, you can use the following command to run RCU in command-line (no GUI) mode
+to create your FMW schemas.  You will need to provide the right prefix and connect string.
+You will be prompted to enter the password for the `sys` user, and then the password to use
+for the regular schema users:
+
+```bash 
+/u01/oracle/oracle_common/bin/rcu \
+  -silent \
+  -createRepository \
+  -databaseType ORACLE \
+  -connectString oracle-db:1521/devpdb.k8s \
+  -dbUser sys \
+  -dbRole sysdba \
+  -useSamePasswordForAllSchemaUsers true \
+  -selectDependentsForComponents true \
+  -schemaPrefix FMW1 \
+  -component MDS \
+  -component IAU \
+  -component IAU_APPEND \
+  -component IAU_VIEWER \
+  -component OPSS  \
+  -component WLS  \
+  -component STB
+```
+
+You need to make sure that you maintain the association between the database schemas and the
+matching domain just like you did in a non-Kubernetes environment.  There is no specific
+functionality provided to help with this.  We recommend that you consider making the RCU
+prefix the same as your `domainUID` to help maintain this association. 
+
+##### Dropping schemas
+
+If you want to drop the schema, you can use a command like this:
+
+```bash
+/u01/oracle/oracle_common/bin/rcu \
+  -silent \
+  -dropRepository \
+  -databaseType ORACLE \
+  -connectString oracle-db:1521/devpdb.k8s \
+  -dbUser sys \
+  -dbRole sysdba \
+  -selectDependentsForComponents true \
+  -schemaPrefix FMW1 \
+  -component MDS \
+  -component IAU \
+  -component IAU_APPEND \
+  -component IAU_VIEWER \
+  -component OPSS \
+  -component WLS \
+  -component STB
+```
+
+Again, you will need to set the right prefix and connection string, and you will be prompted
+to enter the `sys` user password.
+
+#### Create a Kubernetes secret with the RCU credentials
+
+You also need to create a Kubernetes secret containing the credentials for the database schemas.
+When you create your domain using the sample provided below, it will obtain the RCU credentials
+from this secret. 
+
+A [sample](/weblogic-kubernetes-operator/blob/master/kubernetes/samples/scripts/create-rcu-credentials/README.md)
+is provided that demonstrates how to create the secret. 
+
+
+#### Creating a FMW Infrastructure domain
+
+Now that you have your Docker images and you have created your RCU schemas, you are ready
+to create your domain.  A [sample]({{< relref "/samples/simple/domains/fmw-domain/_index.md" >}})
+is provided that demonstrates how to create a FMW Infrastructure domain. 
+
```

Comment:
#### Patching the FMW Infrastructure Image
There are two kind of patches that can be applied to the FMW Infrastructure binaries 1) ZDT compliant, meaning that it can be applied in a rolling fashion and 2) non ZDT compliant patches, meaning that the domain must be shutdown and restarted.  

When you need to apply non-ZDT patches to the FMW Infrastructure binary image this will require that you shutdown the entire domain.  Reference the documentation on [life cycle operations] to the domain ({{< relref " https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/" >}})
An example of none ZDT compliant patches is one that includes a schema change that can not be applied dynamically.  When the schema has changed you need to TODO TODO TODO Mark what do you do when there is a schema change?  Re run RCU and then apply configuration change to the domain?  TODO TODO TODO.
If the patch is ZDT compliant and can be applied in a rolling fashion simply patch the image as in the sample  [sample]({{< relref "https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure/samples/12213-patch" >}}), edit the domain Custom Resource with the name of the new image and the Operator will initiate the rolling restart.
For more information of how to do CI/CD please see https://oracle.github.io/weblogic-kubernetes-operator/userguide/cicd/ [documentation]({{< relref "https://oracle.github.io/weblogic-kubernetes-operator/userguide/cicd/ " >}}).